app/routes/admin/police/add.js

```diff
@@ -0,0 +1,136 @@
+const ConfirmViewModel = require('../../../models/common/confim')
+const { validatePayloadBuilder } = require('../../../schema/common/validatePayload')
+const {
+  isInputFieldInPayload, hasAreYouSureRadioBeenSelected, hasConfirmationFormBeenSubmitted, confirmFlowValidFields,
+  duplicateEntrySchema
+} = require('../../../schema/portal/common/single-submit')
+const { addPoliceForce } = require('../../../api/ddi-index-api/police-forces')
+const { getUser } = require('../../../auth')
+const { PoliceForceAddedViewModel } = require('../../../models/admin/police/builder')
+const { ApiConflictError } = require('../../../errors/api-conflict-error')
+
+const addRemoveConstants = addRemove.policeConstants
+
+const fieldNames = {
+  recordTypeText: addRemoveConstants.messageLabel,
+  recordType: addRemoveConstants.inputField,
+  action: 'add',
+  buttonText: `Add ${addRemoveConstants.messageLabel}`
+}
+
+const stepOneCheckSubmitted = {
+  method: request => {
+    const policeForce = validatePayloadBuilder(isInputFieldInPayload(addRemoveConstants.inputField, addRemoveConstants.messageLabelCapital))(request.payload)
+    return policeForce.police
+  },
+  failAction: (_request, h, error) => {
+    const backLink = addRemoveConstants.backLinks.index.get
+
+    return h.view(views.addAdminRecord, new FormViewModel({
+      backLink,
+      ...fieldNames
+    }, undefined, error)).code(400).takeover()
+  },
+  assign: 'inputField'
+}
+
+const stepTwoCheckConfirmation = {
+  method: request => {
+    return validatePayloadBuilder(hasConfirmationFormBeenSubmitted)(request.payload)
+  },
+  failAction: (request, h) => {
+    const recordValue = request.payload[addRemoveConstants.inputField]
+    const backLink = addRemoveConstants.backLinks.add.get
+
+    return h.view(views.confirm, new ConfirmViewModel({
+      backLink,
+      confirmText: `Are you sure you want to add ‘${recordValue}’ to the Index?`,
+      nameOrReference: addRemoveConstants.inputField,
+      recordValue,
+      action: 'add'
+    })).code(200).takeover()
+  },
+  assign: 'confirmPage'
+}
+
+const stepThreeCheckConfirmation = {
+  method: request => {
+    const { confirm } = validatePayloadBuilder(hasAreYouSureRadioBeenSelected)(request.payload)
+    return confirm
+  },
+  assign: 'addConfirmation',
+  failAction: (request, h, error) => {
+    const recordValue = request.payload[addRemoveConstants.inputField]
+    const backLink = routes.addPoliceForce.get
+
+    return h.view(views.confirm, new ConfirmViewModel({
+      backLink,
+      confirmText: `Are you sure you want to add ‘${recordValue}’ to the Index?`,
+      nameOrReference: addRemoveConstants.inputField,
+      recordValue,
+      action: 'add'
+    }, undefined, error)).code(400).takeover()
+  }
+}
+
+module.exports = [
+  {
+    method: 'GET',
+    path: `${routes.addPoliceForce.get}`,
+    options: {
+      auth: { scope: [admin] },
+      handler: async (_request, h) => {
```

Comment:
Any reason you decided to change 'request' to '_request' ?

Reply:
because it's not being used, it gets around eslint issues as eslint accepts an _ variable if not used.

Reply:
Good point